packages/browser/__tests__/login/oidc/oidcHandlers/LegacyImplicitFlowOIDCHandler.spec.ts

```diff
@@ -35,14 +33,35 @@ import {
   IOidcOptions,
   StorageUtilityMock,
 } from "@inrupt/solid-client-authn-core";
+import URL from "url-parse";
+import { JSONWebKey } from "jose";
+
+const mockJWK = {
+  kty: "EC",
+  kid: "oOArcXxcwvsaG21jAx_D5CHr4BgVCzCEtlfmNFQtU0s",
+  alg: "ES256",
+  crv: "P-256",
+  x: "0dGe_s-urLhD3mpqYqmSXrqUZApVV5ZNxMJXg7Vp-2A",
+  y: "-oMe9gGkpfIrnJ0aiSUHMdjqYVm5ZrGCeQmRKoIIfj8",
+  d: "yR1bCsR7m4hjFCvWo8Jw3OfNR4aiYDAFbBD9nkudJKM",
+};
+
+jest.mock("@inrupt/oidc-dpop-client-browser", () => {
+  return {
+    generateJwkForDpop: async (): Promise<typeof mockJWK> => mockJWK,
+    createDpopHeader: async (
+      _audience: URL,
+      _method: string,
+      _jwt: JSONWebKey
+    ): Promise<string> => "someToken",
```

Comment:
Very minor, but I'd move this string to a constant, e.g. `const mockDpopTokenValue = "Some opaque DPoP token value";`, so that `expect()` calls can refer to that same constant rather than hard-coding values.

Reply:
I would have preferred to do this as well, but I don't know how to do this/if it's possible due to `jest` behaviour: the mocks are hoisted at the top of the file when `jest` runs, which makes any constant in the mock closure uninitialized.